### PR TITLE
[WIP] Replace trafico with github actions

### PR DESCRIPTION
**Pull request by matt-fidd**
From discussion in #2076 

The trafico bot has been down for a little while meaning that PRs aren't being properly labelled, this PR aims to replace the bot with github workflows and expand the functionality to include more automatic statuses.

### CODE REVIEW

**Comment by psybers** [comment]
Just a note that #2895 aims to also fix the issue.

File: .github/workflows/wip.yml
Comment (psybers) [inline .github/workflows/wip.yml:1]: Why was this removed? The idea was to add '[WIP]' to every new PR and allow people to drop it when ready.

Comment (matt-fidd) [reply .github/workflows/wip.yml:1]: It has been replaced with [.github/workflows/pr-opened-add-wip.yml](https://github.com/actualbudget/actual/pull/2900/files#diff-496d4c4ceac6c9e165d6e5966e363d19748deb315f7458388ca561860fe50048)
